frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -0,0 +1,300 @@
+import { useFormikContext, yupToFormErrors } from 'formik';
+import { useApi } from 'hooks/useApi';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import useCodeLookups from 'hooks/useLookupCodes';
+import { noop } from 'lodash';
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import styled from 'styled-components';
+import { InventoryPolicy } from '../components/InventoryPolicy';
+import * as API from 'constants/API';
+import { IParcel } from 'actions/parcelsActions';
+import { ParcelDetailTabs } from 'features/properties/containers/ParcelDetailContainer';
+import { defaultAddressValues } from 'features/properties/components/forms/subforms/AddressForm';
+import { ParcelIdentificationForm } from './subforms/ParcelIdentificationForm';
+import { LandUsageForm } from './subforms/LandUsageForm';
+import { LandReviewPage } from './subforms/LandReviewPage';
+import { defaultPidPinFormValues } from 'features/properties/components/forms/subforms/PidPinForm';
+import { defaultLandValues } from 'features/properties/components/forms/subforms/LandForm';
+import {
+  defaultFinancials,
+  filterEmptyFinancials,
+  IFinancial,
+  IFinancialYear,
+} from 'features/properties/components/forms/subforms/EvaluationForm';
+import { IFormParcel } from 'features/properties/containers/ParcelDetailFormContainer';
+import { EvaluationKeys } from 'constants/evaluationKeys';
+import { FiscalKeys } from 'constants/fiscalKeys';
+import _ from 'lodash';
+import { useDispatch } from 'react-redux';
+import { ParcelSchema } from 'utils/YupSchema';
+import { createParcel, updateParcel } from 'actionCreators/parcelsActionCreator';
+import { LandValuationForm } from './subforms/LandValuationForm';
+
+const Container = styled.div`
+  background-color: #fff;
+  height: 100%;
+  width: 100%;
+  overflow-y: scroll;
+`;
+
+const FormContentWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  margin-bottom: 50px;
+`;
+
+const FormContent = styled.div`
+  border-top: 1px solid #666666;
+  width: 100%;
+  min-height: 100px;
+`;
+
+const FormFooter = styled.div`
+  display: flex;
+  width: 100%;
+  height: 70px;
+  align-items: center;
+`;
+
+const FillRemainingSpace = styled.span`
+  flex: 1 1 auto;
+`;
+
+/**
+ * Create formiks initialValues by stitching together the default values provided by each subform.
+ */
+export const getInitialValues = (): any => {
+  return {
+    ...defaultPidPinFormValues,
+    ...defaultLandValues,
+    ...defaultInformationFormValues,
+    latitude: '',
+    longitude: '',
+    address: defaultAddressValues,
+    buildings: [],
+    financials: defaultFinancials,
+  };
+};
+
+/**
+ * Do an in place conversion of all values to their expected API equivalents (eg. '' => undefined)
+ * @param values the parcel value to convert.
+ */
+export const valuesToApiFormat = (values: ISteppedFormValues<IFormParcel>): IFormParcel => {
```

Comment:
Is there a way we could extract this as a helper function that would know what to do based on an interface or configuration file?